src/Microsoft.Diagnostics.Runtime.Utilities/ClrObjectExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+using Microsoft.Diagnostics.Runtime;
+
+namespace Microsoft.Diagnostics.Runtime.Utilities
+{
+    public static class ClrObjectExtensions
+    {
+        public static string GetStringValue(this ClrObject clrObject, int maxLength = int.MaxValue)
+        {
+            if (clrObject.Type.ElementType != ClrElementType.String) return null;
+            return clrObject.Type.ClrObjectHelpers.ReadString(clrObject.Address, maxLength);
+        }
+
+        public static UInt64 GetGraphSize(this ClrObject clrObject)
```

Comment:
I'm planning to build something this week which will make this not needed.  Can you pause for a day or two?  I'm going to add a function to ClrObject which enumerates the entire graph pointed to by that object.  You should be able to use that with a linq expression to calculate graph size.

Reply:
Sure. I can pause any PR or just this one. I have more code in my own library that could be useful. Just let me know.

BTW in my own library I created new classes to represent a graph (the starting node and the children). This avoids the need to enumerate each ClrObject every time you are working on the graph and the need to check if it has been already visited. If you are making changes to ClrObject we can discuss this separately in order to ease this scenario as well.

Reply:
With regards to the tests, do you have any idea on how to compare the graph size obtained through this (or a new method you are going to implement)?
The `sizeof` returned by `dotnet-dump` is slightly different for reference types only.